app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/SubscriptionController.scala

```diff
@@ -45,6 +47,47 @@ class SubscriptionController @Inject()(cc: ControllerComponents)(implicit ec: Ex
     result
   }
 
+  def updateSubscriptionDetails(id: String): Action[AnyContent] = Action { implicit request =>
+    request.body.asJson.fold[Result] {
+      logger.warn("Could not find JSON in body for subscribe update request")
+      BadRequest
+    } { json =>
+      json
+        .validate[SubscriptionUpdateRequest]
+        .fold[Result](
+          { e =>
+            logger.warn(s"Could not validate subscription update body: $e")
+            BadRequest
+          }, { _ =>
+            val result =
+              EitherT(
+                SubscriptionUpdateProfiles
+                  .updateSubscriptionDetails(id)
+                  .map(_.subscriptionUpdateResponse)
+              ).map(subscriptionResponse => Ok(Json.toJson(subscriptionResponse)))
+                .merge
+                .getOrElse(
+                  Ok(
+                    Json.toJson(
+                      SubscriptionUpdateResponse(
+                        "CGT",
+                        LocalDateTime.of(2015, 12, 17, 9, 30, 47).toString,
+                        "0134567910",
+                        "XXCGTP123456789",
+                        "GB",
```

Comment:
the same here for these verifiers - you could even have the timestamp as a real one as well